src/utils/getTypeString.ts

```diff
@@ -185,14 +188,54 @@ function instanceOfNonRecursivePP(type: TypeNode): boolean {
     type instanceof ImportRefType ||
     type instanceof IntLiteralType ||
     type instanceof IntType ||
-    type instanceof ModuleType ||
```

Comment:
Why is this type not needed any more?

Reply:
I think it was never needed. It is the only occurrence of the type in the codebase as far as I remember. It was moved to the test utils in solc-typed-ast so it won't be available once we remove the typestring parser anyway